### DISCUSSION TITLE
Path PNG photo on rectangle arena in Webots

### DISCUSSION THREAD

**Discussion by mahyahosseinit** [cyberbotics/webots#6683] [General]
i have a png file which is the picturre of a path i drew in photoshop, i made the canva size in photoshop 5000*7000 milimeters, (i changed the scale from pixels to milimeter in photoshop)

now i want to make this photo the path of my rectantagle arena in webots so that my robot will be able to follow the line, i follow the guide below:

1- i make a rectangle arena and i make it 5*7m in size

2- i convert the rectangle arena to a base node

3- in ( solid "rectangle arena" > children > solid "floor" > children > shape > appearance PBRAppearance > baseColorMap ImageTexture > url ) i choose the path of my image

*i followed this tutorial: https://youtu.be/bUo2nphq8IQ?si=cmXdNP4Q7eqiUOaQ

the background of my rectangle arena changes into white (as expected) but i cant find the right scale in textureTransform part, i cant see the path (black line i drew on white background) on the rectangle arena.

i change the scale from 0.0000000001 to 1000000000 but nothing seems to be right, what should i do?

This is the path i wanted to have as my background:
![First path by milm(5in7)](https://github.com/user-attachments/assets/1c466d8f-d9b0-4673-9ad0-5d694de5dfa9)

**Comment by Tianfengchi** [comment]
Try this In textureTransforms, change the cale in to 0.5*0.5
